tests/figures/chart/charts_component.test.ts

```diff
@@ -1168,6 +1163,21 @@ describe("charts", () => {
         expect(checkbox.checked).toBe(true);
       }
     });
+
+    test("dataSetsHaveTitle value is kept when changing to a chart without aggregate option then back again", async () => {
```

Comment:
I don't think the testname is accurate, I don't see why the `aggregate` option matters here :sweat_smile:

Reply:
I think it refers to the test just above which specifically tests the propagation of "aggregated".